app/javascript/react/components/NewGameForm.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react"
+import _ from 'lodash'
+
+import ErrorList from "./ErrorList"
+
+const NewGameForm = props => {
+  const [errors, setErrors] = useState({})
+
+  const [formVals, setFormVals] = useState({
+    name: "",
+    description: "",
+    player_num: ""
+  })
+
+  const handleChange = event => {
+    setFormVals({
+      ...formVals,
+      [event.currentTarget.id]: event.currentTarget.value
+    })
+  }
+
+  const validateForm = () => {
+    let newErrors = {}
+    Object.keys(formVals).forEach((field) => {
+      if(formVals[field].trim() === "") {
+        if(field === "player_num"){
+          field = "Number of players"
+        }
+        newErrors = {
+          ...newErrors,
+          [field]: "is blank"
+        }
+      }
+    })
+    setErrors(newErrors)
+    return _.isEmpty(newErrors)
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    if(validateForm()){
+      props.handleFormSubmit(formVals)
+    }
+  }
+
+  return (
+    <div>
+      <h2>Add a game to our library!</h2>
+      <form onSubmit={handleSubmit}>
+        <ErrorList
+          errors={errors}
+        />
+        <label htmlFor="name">Game Name:</label>
+        <input
+          type="text"
+          name="name"
+          id="name"
+          onChange={handleChange}
+          value={formVals.name}
+          />
+
+        <label htmlFor="description">Game Description:</label>
+        <textarea
+          name="description"
+          id="description"
+          onChange={handleChange}
+          value={formVals.description}
+          />
+
```

Comment:
I would bring the "/>" on line 60 & 68 slightly to the left so they are in line with the opening "<".  For example the `ErrorList` on 50-52 is formatted correctly